docs/pages/technology/vc-concepts.rst

```diff
@@ -13,84 +13,87 @@ Multiple credentials can give a DID its name, the concept mappings makes sure on
+            "city": "<<credentialSubject.organization.city>>"
+          }
         }
-    }
 
-or as key-value:
-
-.. code-block:: bash
-
-    organization.name=X
-    organization.city=Y
-
-Generic fields
-==============
-
-The following template only defines the generic fields. The generic fields are required for each credential. A `type` is also always required.
-
-.. code-block:: json
-
-    {
-        "id": "<<id>>",
-        "issuer": "<<issuer>>",
-        "type": "SomeCredential",
-        "credentialSubject": {
-            "id": "<<subject>>",
-        }
-    }
+It contains 4 main parts: `concept`, `credentialType`, `indices` and `template`.
+The NutsOrganizationCredential is mapped to the `organization` concept.
+When searching for credentials, you use the concept name. When creating a credential you use the credential type.
+When you resolve a credential, you'll always get the raw credential.
 
 Indices
 =======
 
-When searching for or manipulating records in a DB, it's always important to have the correct indices.
-It doesn't really matter which DB you're using, it'll always require some sort of index to be able to perform.
+Each configuration may contain a list of indices. Each index has a name and contains a list of parts.
+The name of the index is used to identify the index. Each index is scoped to its own credential type.
+An index part must contain a `path`. The path is a JSON path query.
+The values found at that location will be used for indexing.
+See [go-leia](https://github.com/nuts-foundation/go-leia) for the syntax and available options.
+
+An index part may contain an `alias`. The alias is used as a search key.
+In the example above, the JSON path `credentialSubject.organization.name` has an alias of `organization.name`.
+This means that in a search query the key `organization.name` can be used.
+Aliases are not unique to a credential type. This allows for searching over multiple credential types with a single query.
 
-In the concept templates, we use the `@T{1_1},{2_1}` syntax for this. The full regex is:
+An index part can also contain a transformer and/or a tokenizer. A tokenizer will split a value into multiple values to be indexed.
+For example: the whitespace tokenizer will split a sentence into a list of words to be indexed.
+The name *healthcare organization the Nutty professor* will be split and can be found by just searching for *healthcare*, *organization*, *the*, *Nutty* or *professor*.
+Possible values for tokenizer: `whitespace`.
 
-.. code-block:: text
+A transformer transforms a value before the value is used as an index key. A transformer will also transform search parameters that use that index.
+For example: the lowercase transformer will transform all indexed values and search params to lowercase.
+The word *Nutty* in the previous example could then be found with *nutty*, *NUTTY* or any other combination of upper- and lowercase letters.
+Possible values for transformer: `lowerCase`, `cologne`.
```

Comment:
is `lowerCase` case sensitive in itself? Maybe `lowercase` is less error prone in that case.

Reply:
did both, suggested to use `lowercase` and comparing after applying `strings.ToLower`